servers.json

```diff
@@ -1,1469 +1,1475 @@
-[ 
-    {
```

Comment:
You completely changed the formatting of servers.json. Undo your formatting changes.